doc/commands/do.8.rst

```diff
@@ -0,0 +1,140 @@
+with the ``@`` character, it is treated as a group ID or a module.
+
+Individual actions and object types are specified by options and are combinable.
+
+Other options have the same meaning and use as with the commands (``install``, ``remove``, ``upgrade``, ``downgrade``,
+and ``reinstall``). The exceptions are the ``--installed-from-repo`` and ``--from-repo`` options. While they have
+the same meaning, they only apply to the packages that follow them on the command line. Additionally, they can be
+repeated with different values.
+
+
+Options
+=======
+
+``--action``
+    | Action to be done on the following items.
+
+    This is a mandatory option that must precede the transaction's items (objects). The do command needs
+    this information to know which action to perform on the specified items.
+    Supports: ``install``, ``remove``, ``upgrade``, ``downgrade``, ``reinstall``.
+
+``--type``
+    | Type of the following items.
+
+    Supports: ``auto``, ``package``, ``group``. ``auto`` is the default until the ``type`` option is used.
+
+``--allowerasing``
+    | Allow removing of installed packages to resolve any potential dependency problems.
+
+``--skip-broken``
+    | Resolve any dependency problems by removing packages that are causing problems from the transaction.
+
+``--skip-unavailable``
+    | Allow skipping packages that are not available in repositories. All available packages will be installed.
+
+``--allow-downgrade``
+    | Enable downgrade of dependencies when resolving the requested operation.
+
+``--no-allow-downgrade``
+    | Disable downgrade of dependencies when resolving the requested operation.
+
+.. include:: ../_shared/options/installed-from-repo.rst
+
+.. include:: ../_shared/options/from-repo.rst
+
+``--downloadonly``
+    | Download the resolved package set without executing an RPM transaction.
+
+``--offline``
+    | Store the transaction to be performed offline. See :manpage:`dnf5-offline(8)`, :ref:`Offline command <offline_command_ref-label>`.
+
+.. include:: ../_shared/options/advisories.rst
+
+.. include:: ../_shared/options/advisory-severities.rst
+
+.. include:: ../_shared/options/bzs.rst
+
+.. include:: ../_shared/options/cves.rst
+
+.. include:: ../_shared/options/security.rst
+
+.. include:: ../_shared/options/bugfix.rst
+
+.. include:: ../_shared/options/enhancement.rst
+
+.. include:: ../_shared/options/newpackage.rst
+
+
+Examples
+========
+
+``dnf5 do --action=install tito``
+    Install the ``tito`` package.
+
+``dnf5 do --action=remove sddm-wayland-plasma --action=install sddm-x11``
+    Replace package ``sddm-wayland-plasma`` with ``sddm-x11``.
+
+``dnf5 do --action=install --type=group id=office 'name=Games and Entertainment' --type=package iftop --action=remove atop``
+    Install the ``office`` group (by ID), the ``Games and Entertainment`` group (by name), and the ``iftop`` package,
+    and, remove the ``atop`` package.
```

Comment:
Please remove the superfluous comma:
`iftop package, and, remove the atop package` -> `iftop package, and remove the atop package`